### ISSUE TITLE
[Content] Review and update the Demo page

### ISSUE THREAD

**Issue by RCheesley** [mautic/low-no-code#20]
This issue has a corresponding Jira ticket here: https://mautic.atlassian.net/browse/WR-177. Ping us in #hacktoberfest if you need inviting to Jira. The content is written below.

You will find the wireframes for the new website at https://www.figma.com/design/OIqb9D6IT3mOP2HjqlZ230/Mautic.org-website-wireframe?node-id=71-267&t=wmMT8GYs1z5xCZuH-1 which shows the kinds of blocks (far right) and layouts we have available. There is one specifically for community, so we need to write the content and decide if this layout will work for what we want to see on the page.

This is an important page as it's the 'front door' for people who want to try Mautic, and also is a trust-building page for folks considering Mautic. We could consider giving a few options like they do on Acquia's website: https://www.acquia.com/request-a-demo/live-demo-form where you can either spin up a trial, watch a clickthrough recorded demo, or schedule a real-time meeting for a demo.

If you'd like to make a mockup on Figma / Canva / Whatever to show the web team how you'd like this page to look (e.g. if you want to use images, layouts etc), please do (and share it with us in the issue comments).

Copied below from the Jira issue:

This issue relates to updating the content for Demo, which you can find at https://www.mautic.org/demo. If there are any comments from the team they will be found below.

The link for writing the updated content can be found at https://docs.google.com/document/d/1exRmNuXsfIDewqtYQhekKZcHm0Tc3XGdM4nJDLdEqTM/edit?usp=drivesdk.

Thanks for your help!

**Comment by Breakthroughbee** [comment]
Hi @RCheesley can I be assigned this issue?

**Comment by RCheesley** [comment]
Absolutely, @Breakthroughbee ! Let us know if you need any guidance!

**Comment by Breakthroughbee** [comment]
Thanks @RCheesley, all i need to do is edit the doc file above with the new content for the demo page, right?

**Comment by RCheesley** [comment]
Yes that's right, please use the Google Doc to propose the content. If you would like to make a mockup on Figma or Canva to visually show what it should look like then please do, but include a link. You can ask questions in #wg-website-rebuild on Slack if you need any guidance from the team who are building the new website!

**Comment by RCheesley** [comment]
Hi @Breakthroughbee great to see you've made a start on the content for the page. Could you perhaps make a bit of a sketch as to how you would see this page looking, using the blocks available on the Figma link? It doesn't have to be pixel perfect but it would help a lot for us to know what you envisage the page looking like - feel free to do it in Canva / Figma or as a sketch on paper which you take a photo of, whatever works. It's also nice if you can share research for other similar pages that you've taken inspiration from. The content does not need to be the same as the current site, in fact we want it to be improved and optimised. Let us know if you have any questions!

**Comment by Breakthroughbee** [comment]
Alright! I Will do that and revert shortly.

**Comment by RCheesley** [comment]
Wonderful, shout if you have any questions!

**Comment by Breakthroughbee** [comment]
@RCheesley I have added a sketch to the doc, with the references I consulted. You can go through it once again.
